core-application/src/main/java/gohigher/application/service/ApplicationQueryService.java

```diff
@@ -21,18 +24,34 @@ public class ApplicationQueryService implements ApplicationQueryPort {
 	private final ApplicationPersistenceQueryPort applicationPersistenceQueryPort;
 
 	@Override
-	public List<CalenderApplicationResponse> findByMonth(CalenderApplicationRequest request) {
-		List<Application> applications = applicationPersistenceQueryPort.findByIdAndMonth(request.getUserId(),
+	public List<CalendarApplicationResponse> findByMonth(CalendarApplicationRequest request) {
+		List<Application> applications = applicationPersistenceQueryPort.findByUserIdAndMonth(request.getUserId(),
 			request.getYear(), request.getMonth());
 
 		return applications.stream()
-			.flatMap(ApplicationQueryService::extractCalenderResponses)
+			.flatMap(this::extractCalendarResponses)
 			.toList();
 	}
 
-	private static Stream<CalenderApplicationResponse> extractCalenderResponses(Application application) {
+	@Override
+	public List<DateApplicationResponse> findByDate(DateApplicationRequest request) {
+		List<Application> applications = applicationPersistenceQueryPort.findByUserIdAndDate(request.getUserId(),
+			request.getDate());
+		return applications.stream()
+			.flatMap(this::extractDateApplicationResponses)
+			.toList();
+	}
+
+	private Stream<CalendarApplicationResponse> extractCalendarResponses(Application application) {
 		return application.getProcesses().stream()
-			.map(process -> new CalenderApplicationResponse(application.getId(), process.getId(),
+			.map(process -> new CalendarApplicationResponse(application.getId(), process.getId(),
 				application.getCompanyName(), process.getType().name(), process.getSchedule()));
 	}
+
+	private Stream<DateApplicationResponse> extractDateApplicationResponses(Application application) {
+		return application.getProcesses().stream()
+			.map(ProcessResponse::from)
+			.map(processResponse -> new DateApplicationResponse(application.getId(), application.getCompanyName(),
+				processResponse));
```

Comment:
```suggestion
	private Stream<DateApplicationResponse> extractDateApplicationResponses(Application application) {
		return application.getProcesses().stream()
			.map(ProcessResponse::from)
			.map(response -> DateApplicationResponse.of(application.getId(), application.getCompanyName(), response));
```
바로 윗 줄의 코드와 같이 DTO 내부로 숨기는건 어떤가요?

Reply:
생성자와 팩토리 메서드의 차이이기에 큰 변화를 느끼기는 어려운거 같아요!
그래서 말씀해주신 `DateApplicationResponse.of(application,process)`의 형태로 변경하고 내부에서 필요한 필드들을 get하는 방식으로 변경하였습니다!